### DISCUSSION TITLE
Acoustic guitar separation

### DISCUSSION THREAD

**Discussion by edgarcin95** [asteroid-team/asteroid#593] [Q&A] [answered]
Hello, I am searching for a good toolkit-library for my project. I want to create a deep learning model that separates an acoustic guitar from a song, so I create my own dataset including the WAV files of the mix and the acoustic guitar for several songs. Is it possible to load my dataset for training in Asteroid? Also I would like to build custom architectures.

**Comment by mpariente** [comment, marked as answer]
Hello, 

Yes, it is possible. You need to write a custom PyTorch Dataset, and your architecture can use Asteroid's building blocks !
